rtcpeerconnection.go

```diff
@@ -158,7 +158,9 @@ func New(configuration RTCConfiguration) (*RTCPeerConnection, error) {
 		}
 	}
 
-	pc.networkManager, err = network.NewManager(urls, pc.generateChannel, pc.iceStateChange, defaultSettingEngine.EphemeralUDP.PortMin, defaultSettingEngine.EphemeralUDP.PortMax)
+	pc.networkManager, err = network.NewManager(urls, pc.generateChannel, pc.iceStateChange,
```

Comment:
Can we just pass the `ice.AgentConfig` directly here? Having lots of parameters is hard to read (and easy to get them out of order by accident)

Reply:
It's public, I don't see why not